common/religion/religions/shamanism_group.txt

```diff
@@ -619,5 +619,67 @@
 			#Magic
 			doctrine = doctrine_life_magic_accepted
 		}
+
+		#Trogg religion
+		trogg_religion = {
+			#Main Group
+			color = rgb { 82 58 40 }
+			icon = wczu_trogg_religion
+			reformed_icon = wczu_trogg_religion
+
+			doctrine = unreformed_faith_doctrine
+
+			holy_site = ulduar #Where the Trogg were created
+			holy_site = red_desert # Uldaman, where many Trogg were held in stasis before the dwarves dug them out
+
+			#Not sure about these ones
+			holy_site = ironforge #Because the trogg want to kill the dwarves
+			holy_site = gnomeregan  #Because the trogg were disturbed by the Gnomes and took the city
+			holy_site = alterac_valley #Trogg infest a mine here
+
+			#Main Group
+			doctrine = doctrine_no_head
+
+			doctrine = tenet_natural_primitivism
+			doctrine = tenet_sanctity_of_nature
+			doctrine = tenet_human_sacrifice
+
+			#Magic
+			#Ignore all except some shaman, they don't know any other magic
+			doctrine = doctrine_shadow_magic_ignored
+			doctrine = doctrine_disorder_magic_ignored
+			doctrine = doctrine_death_magic_ignored
+			doctrine = doctrine_elemental_decay_magic_accepted
+			doctrine = doctrine_elemental_water_magic_ignored
+			doctrine = doctrine_elemental_air_magic_ignored
+			doctrine = doctrine_elemental_earth_magic_approved
+			doctrine = doctrine_elemental_spirit_magic_accepted
+		}
+
+		#Drogbar religion
+		drogbar_religion = {
+			#Main Group
+			color = rgb { 117 72 39 }
+			icon = wczu_drogbar_religion
+			reformed_icon = wczu_drogbar_religion
+
+			doctrine = unreformed_faith_doctrine
+			
+			holy_site = icemist #These are placeholders, can't put proper ones until Broken Isles are mapped
+			holy_site = taunkale
+			holy_site = wintergarde
+			holy_site = njord
+			holy_site = grizzlemaw
+
+			#Main Group
+			doctrine = doctrine_no_head
+
+			doctrine = tenet_natural_primitivism
+			doctrine = tenet_sanctity_of_nature
+			doctrine = tenet_ancestor_worship
+
+			#Magic
+			doctrine = doctrine_elemental_earth_magic_approved #Drogbar have strong control over earth
```

Comment:
I propose these laws for both faiths:
Shadow - Ignored or Shunned, they're primitive and don't care for bad magic use
Fel - Ignored or Shunned, they're primitive and don't care for magic use
Death - Ignored or Shunned, they're primitive and don't care for magic use
Decay - Approved, they don't worship elements thus don't mind enslaving them
Fire - Accepted, troggs use it and they live close to lava and stuff.
Air - Ignored, their counter-magic.
Earth - Idolised... well, you know.